src/corelibs/U2Lang/src/support/SimpleWorkflowTask.cpp

```diff
@@ -148,6 +149,14 @@ SimpleMSAWorkflow4GObjectTask::SimpleMSAWorkflow4GObjectTask(const QString &task
     MultipleSequenceAlignment al = MSAUtils::createCopyWithIndexedRowNames(msaObjectPointer->getMultipleAlignment());
 
     MultipleSequenceAlignmentObject *msaObject = MultipleSequenceAlignmentImporter::createAlignment(msaObjectPointer->getEntityRef().dbiRef, al, os);
+    for (const MultipleSequenceAlignmentRow &row : al->getMsaRows()) {
+        if (row->getData().contains("*")) {
+            msaObject->replaceAllCharacters('*', 'X');
```

Comment:
'X' has semantic meaning (unknown amino acid) for protein alignment.
'N' has the same meaning (unknown nucleotide) for DNA alignment.
Why did you select 'X' but not 'N' or something else here?